ksql-engine/src/test/java/io/confluent/ksql/codegen/SqlToJavaVisitorTest.java

```diff
@@ -26,12 +30,18 @@
 
   private MetaStore metaStore;
   private Schema schema;
-  private InternalFunctionRegistry functionRegistry;
+  private final InternalFunctionRegistry functionRegistry = new InternalFunctionRegistry();
 
   @Before
   public void init() {
-    metaStore = MetaStoreFixture.getNewMetaStore(new InternalFunctionRegistry());
-    functionRegistry = new InternalFunctionRegistry();
+    metaStore = MetaStoreFixture.getNewMetaStore(functionRegistry);
+    // load udfs that are not hardcoded
+    new UdfLoader(metaStore,
```

Comment:
This is being repeated - maybe move to some test util class?

Reply:
already did